### PR TITLE
feat: wishlist

### PR DESCRIPTION
**Pull request by shashidwivedi**
Add wishlist page with wishlist management

- option to add to cart
- remove from wishlist

127 0 0 1_5500_pages_wishlist html

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700;900&amp;family=Poppins:wght@300;400;600;700;900&amp;display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
```
Comment (nishasen) [inline pages/wishlist.html:24]: try to add common links in single file

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      </div>
+    </header>
+    <main class="main">
+      <div class="wishlist">
+        <h2 class="wishlist-heading heading-3">Wishlist</h2>
+        <div class="flex-row">
+          <div class="wishlist-item">
+            <div class="card card_with-shadow">
+              <div class="card-content">
+                <img
```
Comment (nishasen) [inline pages/wishlist.html:58]: add alt attribute to img

File: pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                  class="card-media"
+                  src="../images/jonathan-borba-_peqvnRW7tc-unsplash.jpg"
```
Comment (nishasen) [inline pages/wishlist.html:60]: add alt attribute to img

**Review by nishasen — APPROVED** [review]
Added few changes, else everything looks good.

File: pages/wishlist.html (hunk 1 above)
Comment (shashidwivedi) [reply pages/wishlist.html:24]: Yes Nisha, will refactor later

**Review by Gautam2305 — APPROVED** [review]
use the alt attribute as nishasen reviewed....rest all looks good

**Review by Sameony — APPROVED** [review]
Nice code, do try to keep the head tag cleaner

File: pages/wishlist.html (hunk 2 above)
Comment (shashidwivedi) [reply pages/wishlist.html:58]: Yes, will do that as part of refactor

**Pull request merged**